xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-macros/xwiki-platform-rendering-macro-code/pom.xml

```diff
@@ -52,7 +52,7 @@
     <dependency>
       <groupId>org.pygments</groupId>
       <artifactId>pygments</artifactId>
-      <version>2.5.1</version>
+      <version>2.4.2</version>
```

Comment:
I'd put a comment and a link to the explanation to why we should upgrade to 2.5+ (2 reasons)